packaging/templates/mainnet-v1/without-sentry/bor/pbss_config.toml

```diff
@@ -0,0 +1,179 @@
+# NOTE: Update and uncomment: `keystore`, `password`, and `unlock` fields.
+
```

Comment:
PBSS are a great addition, Dan !

curious, we want our mesh nodes to be high-bandwidth block relayers. in your opinion, that being the case, do you recommend we use this config without-sentry ?

Reply:
You would use just the sentry config package generated at the build. You can then either modify with ansible or suggest other changes and we can add